schema/samcli.json

```diff
@@ -143,9 +143,175 @@
+                },
+                "use_json": {
+                  "title": "use_json",
+                  "type": "boolean",
+                  "description": "Indicates whether to use JSON as the format for the output AWS CloudFormation template. YAML is used by default."
+                },
+                "resolve_s3": {
+                  "title": "resolve_s3",
+                  "type": "boolean",
+                  "description": "Automatically resolve AWS S3 bucket for non-guided deployments. Enabling this option will also create a managed default AWS S3 bucket for you. If one does not provide a --s3-bucket value, the managed bucket will be used. Do not use --guided with this option."
+                },
+                "resolve_image_repos": {
+                  "title": "resolve_image_repos",
+                  "type": "boolean",
+                  "description": "Automatically create and delete ECR repositories for image-based functions in non-guided deployments. A companion stack containing ECR repos for each function will be deployed along with the template stack. Automatically created image repositories will be deleted if the corresponding functions are removed."
+                },
+                "metadata": {
+                  "title": "metadata",
+                  "type": "string",
+                  "description": "Map of metadata to attach to ALL the artifacts that are referenced in the template."
+                },
+                "notification_arns": {
+                  "title": "notification_arns",
+                  "type": "array",
+                  "description": "ARNs of SNS topics that AWS Cloudformation associates with the stack."
+                },
+                "tags": {
+                  "title": "tags",
+                  "type": "string",
+                  "description": "List of tags to associate with the stack."
+                },
+                "parameter_overrides": {
+                  "title": "parameter_overrides",
+                  "type": "string",
+                  "description": "String that contains AWS CloudFormation parameter overrides encoded as key=value pairs."
+                },
+                "signing_profiles": {
+                  "title": "signing_profiles",
+                  "type": "string",
+                  "description": "A string that contains Code Sign configuration parameters as FunctionOrLayerNameToSign=SigningProfileName:SigningProfileOwner Since signing profile owner is optional, it could also be written as FunctionOrLayerNameToSign=SigningProfileName"
+                },
+                "no_progressbar": {
+                  "title": "no_progressbar",
+                  "type": "boolean",
+                  "description": "Does not showcase a progress bar when uploading artifacts to S3 and pushing docker images to ECR"
+                },
+                "capabilities": {
+                  "title": "capabilities",
+                  "type": "array",
```

Comment:
can we set items section to the schema here to let customer knows that the values of the list items should be string?